modules/storages/app/common/storages/peripherals/storage_interaction/nextcloud/file_info_query.rb

```diff
@@ -28,107 +28,122 @@
 # See COPYRIGHT and LICENSE files for more details.
 #++
 
-module Storages::Peripherals::StorageInteraction::Nextcloud
-  class FileInfoQuery
-    using Storages::Peripherals::ServiceResultRefinements
-
-    FILE_INFO_PATH = "ocs/v1.php/apps/integration_openproject/fileinfo"
-    Auth = ::Storages::Peripherals::StorageInteraction::Authentication
-
-    def self.call(storage:, auth_strategy:, file_id:)
-      new(storage).call(auth_strategy:, file_id:)
-    end
-
-    def initialize(storage)
-      @storage = storage
-    end
-
-    def call(auth_strategy:, file_id:)
-      http_options = Util.ocs_api_request.deep_merge(Util.accept_json)
-      Auth[auth_strategy].call(storage: @storage, http_options:) do |http|
-        file_info(http, file_id).map(&parse_json) >> handle_failure >> create_storage_file_info
-      end
-    end
-
-    private
-
-    def file_info(http, file_id)
-      response = http.get(Util.join_uri_path(@storage.uri, FILE_INFO_PATH, file_id))
-      error_data = Storages::StorageErrorData.new(source: self.class, payload: response)
-
-      case response
-      in { status: 200..299 }
-        ServiceResult.success(result: response.body)
-      in { status: 404 }
-        Util.error(:not_found, "Outbound request destination not found!", error_data)
-      in { status: 401 }
-        Util.error(:unauthorized, "Outbound request not authorized!", error_data)
-      else
-        Util.error(:error, "Outbound request failed!", error_data)
-      end
-    end
-
-    def parse_json
-      ->(response_body) do
-        # rubocop:disable Style/OpenStructUse
-        JSON.parse(response_body, object_class: OpenStruct)
-        # rubocop:enable Style/OpenStructUse
-      end
-    end
-
-    def handle_failure
-      ->(response_object) do
-        error_data = Storages::StorageErrorData.new(source: self.class, payload: response_object)
-
-        case response_object.ocs.data.statuscode
-        when 200..299
-          ServiceResult.success(result: response_object)
-        when 403
-          Util.error(:forbidden, "Access to storage file forbidden!", error_data)
-        when 404
-          Util.error(:not_found, "Storage file not found!", error_data)
-        else
-          Util.error(:error, "Outbound request failed!", error_data)
+module Storages
```

Comment:
It could have been written like that to have the same meaning you want and not changing indentation for whole file:
```
module Storages; module Peripherals; module StorageInteraction; module Nextcloud
  class FileInfoQuery
  ...
  end
 end; end; end; end
```

Reply:
right, sadly the autoformatter I use has no rules to cope with that ;)